### PR TITLE
💄 Style Global CSS

### LINKED ISSUE #4: Style Global CSS
- [x] Set default colors
- [x] Set default fonts and font-style

### PR DESCRIPTION
**Pull request by ShiningElektra**
Fixes #4
- Set default colors
- Set default font-style

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
+
+  /* button style */
+  --primary-button: #0bc717;
+  --secondary-button: #109f1c;
+  --tertiary-button: #81e187;
+  --tertiary-button-bg: #d2ffd5;
+
+  /* font-style */
+  --primary-font-size: 45px;
+  --secondary-font-size: 30px;
```
Comment (phlgr) [inline styles/globals.css:17]: This is quite big, is there a reason for it?

**Review by phlgr — COMMENTED** [review]
🚀

File: styles/globals.css (hunk 1 above)
Comment (ShiningElektra) [reply styles/globals.css:17]: It´s only the  button font-size.
But it is still in progress.

Comment (phlgr) [reply styles/globals.css:17]: I still wouldn't declare these here and also not in pixels. I'd rather set a "main"-`font-size` and declare these variables with rem

**Pull request merged**